### ISSUE TITLE
ModuleNotFoundError: No module named 'selective_scan_cuda_oflex'

### ISSUE THREAD

**Issue by AliasChenYi** [MzeroMiko/VMamba#203]
Can you help me? How can I solve it

**Comment by BranStarkkk** [comment]
I think that you should reinstall the selective_scan with the order "cd kernels/selective_scan && pip install ." until "Successfully built selective_scan
Installing collected packages: selective_scan
Successfully installed selective_scan-0.0.2"

**Comment by AliasChenYi** [comment]
Yes, I have done this by following the steps you described, but it is still not resolved.




---Original---
From: ***@***.***&gt;
Date: Wed, May 22, 2024 11:17 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [MzeroMiko/VMamba] ModuleNotFoundError: No module named'selective_scan_cuda_oflex' (Issue #203)




 
I think that you should reinstall the selective_scan with the order "cd kernels/selective_scan &amp;&amp; pip install ." until "Successfully built selective_scan
 Installing collected packages: selective_scan
 Successfully installed selective_scan-0.0.2"

**Comment by BranStarkkk** [comment]
oh... Perhaps you can then install a 'mamba_ssm' and the problem will be solved

**Comment by BranStarkkk** [comment]
oh... Perhaps you can then install a 'mamba_ssm' and the problem will be solved

**Comment by AliasChenYi** [comment]
I have also installed the package of mamba_ssm.




---Original---
From: ***@***.***&gt;
Date: Wed, May 22, 2024 11:55 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [MzeroMiko/VMamba] ModuleNotFoundError: No module named'selective_scan_cuda_oflex' (Issue #203)




 
oh... Perhaps you can then install a 'mamba_ssm' and the problem will be solved

**Comment by AliasChenYi** [comment]
> oh... Perhaps you can then install a 'mamba_ssm' and the problem will be solved

One of my current situations is that I have selective_scan_cuda_core, but no selective_scan_cuda_oflex

**Comment by bryanbocao** [comment]
> oh... Perhaps you can then install a 'mamba_ssm' and the problem will be solved

I also tried ```pip install mamba-ssm``` but the problem of ```https://github.com/MzeroMiko/VMamba/issues/55``` still persists.

**Comment by YanWang-WHPU** [comment]
> Can you help me? How can I solve it

我安装 selctive_scan 0.0.2就解决了，我之前是0,0,1

**Comment by rrryan2016** [comment]
> I have also installed the package of mamba_ssm.
> […](#)

Same to me. Did you resolve it?
